Controller/Payment/Confirm.php

```diff
@@ -135,13 +150,25 @@ public function execute()
             );
             $_order->addCommentToStatusHistory($e->getMessage());
             $_order->save();
+            $this->errorTracker->logErrorToAffirm(
+                transaction_step: 'pre_auth',
+                error_type: ErrorTracker::INTERNAL_SERVER_ERROR,
+                exception: $e,
```

Comment:
I think you will have to convert this to json since you function expects this to be a string.